### ISSUE TITLE
[0.9.0.0 staging - 1756] Law / Migration Citizen no longer option as user pointer

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#18078]
1742 build ( loaded world in that version )

![Screenshot_183](https://user-images.githubusercontent.com/24981362/92410191-a1ed2e80-f143-11ea-8287-44677e3c532e.jpg)
Here i can select "Citizen" as an option and it reads / calculates information correctly

1742 build ( loaded world -> migrated -> 1756 )
![Screenshot_259](https://user-images.githubusercontent.com/24981362/92410073-33a86c00-f143-11ea-8aba-51480e1eb549.jpg)
I can no longer select anything as user specification it returns null value ( <missing> ) in the description field and the button is greyed out as if the array contains no data.

1756 build ( fresh new world )
![Screenshot_260](https://user-images.githubusercontent.com/24981362/92410075-35722f80-f143-11ea-9f80-39424f873ada.jpg)
Here i can select only specific user NOT "Citizen" anymore .

Which is why my conclusion is that Citizen option is moved, broken or missing that is causing the old migrated server to look for a input that it can no longer find making it invalid and locks out the option.

**Comment by beastyn** [comment]
@IcedForge did you have such issues with the new laws?

It was perfectly fine for me on some kind of fresh world (post 1756).

**Comment by IcedForge** [comment]
Yes it works fine with post 1756 build servers, tried a few of my other old worlds and had the same resulting issue.
Tried 3 different worlds and they all worked perfectly fine to generate, and apply the same type law that i originally wanted to try.
So a pointer change between 1742 -> 1756 is causing this issue in the "the checked user" aka "Citizen" being looked for in a faulty location hence why its saying <missing> in the description.
